### DISCUSSION TITLE
difference between doc results and my results

### DISCUSSION THREAD

**Discussion by benoitpaillard** [mdolab/dafoam#583] [Tutorials]
Hi all, when running the multi element optimization from docker with
`preProcessing.sh` and  `runScript.py
`
(`runScript_v2.py` crashes)
I get this:

https://github.com/mdolab/dafoam/assets/6714189/ff66e368-5237-4cab-b5e2-4252e7080d95

Which is quite different from what is shown in the [doc](https://dafoam.github.io/mydoc_tutorials_aero_30n30p.html)

**Comment by friedenhe** [comment]
Thanks for the feedback. We are looking into this problem. If you ran the case without any change, it may be caused by a bug in the run scripts.
